Prometheus.HttpExporter.AspNetCore/HttpRequestCount/HttpRequestCountOptions.cs

```diff
@@ -0,0 +1,11 @@
+namespace Prometheus.HttpExporter.AspNetCore.HttpRequestCount
+{
+    public class HttpRequestCountOptions : HttpExporterOptionsBase
+    {
+        public Counter Counter { get; set; } =
+            Metrics.CreateCounter(DefaultName, DefaultHelp, "code", "method", "action", "controller");
+        
+        private const string DefaultName = "aspnet_http_request_count";
```

Comment:
The standard style is to name "total count" counters with `_total` suffix, so `aspnet_http_requests_total` would be better here.